src/cron/utils/read-markdown-messages.ts

```diff
@@ -23,34 +23,42 @@ const getMarkdownFiles = (directory) => {
       }
     });
   } catch (error) {
-    console.error(`Error reading the directory "${directory}":`, error.message);
+    console.error(
+      `Error reading the directory "${directory}":`,
+      (error as Error).message
+    );
   }
 
   return markdownFiles;
 };
 
 /**
  * Function to read the content of a Markdown file.
- * @param {string} filePath - Path to the file to read.
- * @returns {string|null} - File content or null if an error occurs.
+ * @param filePath - Path to the file to read.
+ * @returns File content or null if an error occurs.
  */
-const readMarkdownFile = (filePath) => {
+const readMarkdownFile = (filePath: string): string | null => {
   try {
     return fs.readFileSync(filePath, 'utf8');
   } catch (error) {
-    console.error(`Error reading the file "${filePath}":`, error.message);
+    console.error(`Error reading the file "${filePath}":`, (error as Error).message);
     return null;
   }
 };
 
+interface ParsedSchedule {
+  variables: Record<string, string>;
```

Comment:
I think here you are defining parameters of parseMarkdownSchedule, and `variables` is not a parameter of this function